clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -1512,6 +1512,43 @@ public void testDescribeConsumerGroupsWithAuthorizedOperationsOmitted() throws E
         }
     }
 
+    @Test
+    public void testDescribeNonConsumerGroups() throws Exception {
+        final HashMap<Integer, Node> nodes = new HashMap<>();
```

Comment:
Lots of duplication in these tests. No need to do it in this PR but I'm wondering if wrapping all the
```
 final HashMap<Integer, Node> nodes = new HashMap<>();
        nodes.put(0, new Node(0, "localhost", 8121));

        final Cluster cluster =
            new Cluster(
                "mockClusterId",
                nodes.values(),
                Collections.<PartitionInfo>emptyList(),
                Collections.<String>emptySet(),
                Collections.<String>emptySet(), nodes.get(0));

        try (AdminClientUnitTestEnv env = new AdminClientUnitTestEnv(cluster)) {
```
boilerplate into a single method that accepts a callable would look better in this test class

Reply:
Totally. If you don't mind, I will tackle this in a separate PR. I'd like to keep this one focus on the bugfix.